### PR TITLE
add place type attraction

### PR DESCRIPTION
**Pull request by alisa911**

### CODE REVIEW

File: openplacereviews-boot/src/main/resources/bootstrap/opr-0-web-bot.json [hunk 1]
```diff
@@ -27,6 +27,11 @@
       			"key" : "tourism",
       			"values" : ["hotel","motel","hostel","apartment","guest_house"],
             	"type" : ["node", "way"]
+          },
+          "attraction": {
+            "key": "historic",
+            "values": ["ruins"],
```
Comment (vshcherb) [inline openplacereviews-boot/src/main/resources/bootstrap/opr-0-web-bot.json]: probably we should add much more attraction list + tourism. we can double check what's supported by OsmAnd tourism category

Comment (davidoe) [reply openplacereviews-boot/src/main/resources/bootstrap/opr-0-web-bot.json]: tourism=camp_site and tourism=caravan_site would be nice

**Comment by amenk** [comment]
@vshcherb what the inclusion strategy to for adding new places? I think there are tons of place types missing which are worth including. (why not all OSM tags?) I always thought this is a limitation at the beginning to keep the database smaller or something like that.

We would be really interested in adding more place types - but the question is if it will be merged or if there is some formal guide line on what to include and what not.

**Comment by vshcherb** [comment]
We need to create proper categories, there is no such thing as all places + it will require moderation (maintaing permanent opr id for each place). 

Counting in OsmAnd ```poi_type``` (https://github.com/osmandapp/OsmAnd-resources/blob/master/poi/poi_types.xml#L1296) there are more than 1000 places, so it's a bit tricky :-)

**Comment by amenk** [comment]
But then we need moderation guidelines :-) Or just include all from OsmAnd ... in a bit more categorized form

File: openplacereviews-boot/src/main/resources/bootstrap/opr-0-web-bot.json (hunk 1 above)
Comment (scaidermern) [reply openplacereviews-boot/src/main/resources/bootstrap/opr-0-web-bot.json]: Also `historic=castle`, `historic=aqueduct`, `historic=church` etc.

**Pull request merged**